src/CppUTest/TestRegistry.cpp

```diff
@@ -50,6 +50,7 @@ void TestRegistry::runAllTests(TestResult& result)
     result.testsStarted();
     for (UtestShell *test = tests_; test != NULL; test = test->getNext()) {
```

Comment:
yeah I agree. 
`UtestShell::isRunIgnore()` is not called outside the class, I will remove it. 
For the `setRunIgnored ` and that member variable, I also think defined in UtestShell is not the best design, but due to current class hierarchy(UtestShell and IgnoreTestShell) and inter-class dependency(TestRegistry and UtestShell Interface), we can not get bettern place to hold the `setRunIgnored ` and the member variable, unless we do some refactorying on the class hierrachy, eg. segregate the UtestShell interface. But currently it's not deserved to refactorying the interface. We can do this as the interface beome fatter.